docs/examples/evaluation/multi_modal/multi_modal_rag_evaluation.ipynb

```diff
@@ -0,0 +1,1242 @@
+{
```

Comment:
<p>let's write down some takeaways. Is this showing that retrieving image-only is sufficient? </p><p> </p><p>also this is a tangent, but I wonder how much of the knowledge is memorized in the generator LLM already (since it's pretty generic world knowledge). Maybe not for this notebook, I'm personally curious if you just give no context, whether if the generator can still do well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>